src/viur/core/bones/file.py

```diff
@@ -226,7 +226,10 @@ def postSavedHandler(self, skel, boneName, key):
         the derived files directly.
         """
         super().postSavedHandler(skel, boneName, key)
-        if current.request.get().is_deferred and current.request_data.get().get("__update_relations_bone") == "derived":
+        if (
+            current.request.get().is_deferred
+            and current.request_data.get().get("__update_relations_bones") == ["derived"]
```

Comment:
What if "derived" is in the set, additionally? 
```suggestion
            and "derived" in (current.request_data.get().get("__update_relations_bones") or ())
```